docs/guides/dependency-installation.rst

```diff
@@ -61,7 +62,7 @@ For illustration, the following example specifies a task that imports its source
     import orquestra.sdk as sdk
 
     @sdk.task(
-        source_import = sdk.GitHubImport(
+        source_import = sdk.GithubImport(
```

Comment:
**suggestion**: did you consider turning these snippets into tests so we don't miss similar errors in the future?

Reply:
Should we get this update out first and do follow ups like that as part of https://zapatacomputing.atlassian.net/browse/ORQSDK-855?